examples/VLCB_1in1out/VLCB_1in1out.ino

```diff
@@ -50,12 +49,11 @@ VLCB::SerialUserInterface serialUserInterface(&can2515);
 VLCB::MinimumNodeService mnService;
 VLCB::CanService canService(&can2515);
 VLCB::NodeVariableService nvService;
-VLCB::ConsumeOwnEventsService coeService;
 VLCB::EventConsumerService ecService;
 VLCB::EventTeachingService etService;
 VLCB::EventProducerService epService;
-VLCB::Controller controller(&modconfig,
-                            {&mnService, &ledUserInterface, &serialUserInterface, &canService, &nvService, &ecService, &epService, &etService, &coeService}); // Controller object
+VLCB::Controller controller(&modconfig, 
+                            { &mnService, &ledUserInterface, &serialUserInterface, &canService, &nvService, &ecService, &epService, &etService }); // Controller object
```

Comment:
Why have you taken out coeService? 
I actually use it for my own testing,

Reply:
Maybe I'm not as with it today as I thought I was.  I'll amend with anything else from the comments.